index.d.ts

```diff
@@ -88,6 +90,46 @@ export interface Options {
 	```
 	*/
 	readonly maximumFractionDigits?: number;
+
+	/**
+	The number of spaces to put between the number and unit.
+
+	If it is not set, the default value is to display 1 space between the number and unit.
+
+	 If it is set to a negative number, the number of spaces between the number and unit will be 0.
+
+	@default 1
+
+	@example
+	```
+	import prettyBytes from 'pretty-bytes';
+	prettyBytes(1920, {spaces: 0});
+	//=> '1.9kB'
+
+	prettyBytes(1920);
+	//=> '1.92 kB'
+	```
+	*/
+	readonly spaces?: number;
+
+	/**
+	Display an uppercase K for units with the "kilo" prefix.
```

Comment:
This is wrong though. When using the uppercase `K`, it refers to KiBi bytes, which is 1024, while this package works on SI units, which is 1000.

https://en.wikipedia.org/wiki/Kilobyte

Reply:
Ah, true. Alright, then I guess this option is kinda pointless in this form.

What I could do, and I don't know the usefulness of it, is when this option (renamed of course to something more sensible) and `binary` is both set, then it'd print `KB` instead of `kiB`.

A more robust way would be to mimic what is on the wiki page's Binary -> Memory column:
![image](https://user-images.githubusercontent.com/29164245/214515258-72ae13aa-0447-4a42-83c7-96fe3199b9f4.png)
By using the decimal units for the binary case, but with KB instead of kB. In this case, I'd name this option `legacyBinaryUnits`, with a default value of `false`. WDYT?

Reply:
I would prefer not including an option like this. I don't really see why anyone should use this option, since it causes ambiguity (`1 MB`, is it 1000 kB or 1024 KiB?).

Reply:
Alrighty, I'll remove it!